packages/react-core/src/components/Toolbar/ToolbarToggleGroup.tsx

```diff
@@ -149,7 +149,7 @@ class ToolbarToggleGroup extends React.Component<ToolbarToggleGroupProps> {
                           aria-label="Show Filters"
                           {...(_isExpanded && { 'aria-expanded': true })}
                           aria-haspopup={_isExpanded && this.isContentPopup()}
-                          aria-controls={expandableContentId}
+                          aria-controls={_isExpanded ? expandableContentId : undefined}
```

Comment:
As an FYI, Jenny and I discussed this and I opened https://github.com/patternfly/patternfly-react/issues/9827 as an investigation as to whether this should be a temporary fix or not